app/controllers/forms_controller.rb

```diff
@@ -25,7 +25,7 @@ def submit
     user = @current_user || create_user
     data = params.require :responses
     FtFormsMailer.send_form(@form, data).deliver_now
-    FtFormsMailer.send_confirmation(user, data).deliver_now
+    FtFormsMailer.send_confirmation(user, data, @form.reply_to).deliver_now
```

Comment:
Since the hidden field puts the reply_to value in the form, the controller should be finding it in the params.